Solaris Github Action Is Failing On Vagrant Install

The Github action, that builds solaris via vagrant is failing, due to errors when installing vagrant from the repositories..

```
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/libv/libvirt/libvirt0_8.0.0-1ubuntu7.9_amd64.deb 404 Not Found [IP: 52.252.75.106 80]
302E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
303Fetched 82.1 MB in 5s (17.0 MB/s)
304Error: Process completed with exit code 100.
```

--- a/.github/workflows/build_vagrant.yml
+++ b/.github/workflows/build_vagrant.yml
@@ -24,6 +24,9 @@ jobs:
 
     - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
+    - name: Update Repos
+      run: sudo apt-get update
+      
     - name: Install Ansible
       run: sudo apt-get install ansible
 

